_sass/components/_main-content.scss

```diff
@@ -97,6 +97,15 @@ $site-top:           100px;
 .va-rr-sg-header--branding {
   background-color: $va-rr-brand;
 }
+.usa-content {
```

Comment:
Remove lines 100 - 107, and add them to the `.va-rr-sg-description` declaration block.